wolfcrypt/src/wc_port.c

```diff
@@ -126,6 +127,161 @@ int wolfCrypt_Cleanup(void)
     return ret;
 }
 
+#if !defined(NO_FILESYSTEM)
+
+int wc_ChangeDir(const char* path)
+{
+    int ret;
+#ifdef USE_WINDOWS_API
+    ret = SetCurrentDirectoryA(path);
+#else
+    ret = chdir(path);
```

Comment:
chdir() wasn't in wolfSSL proper before.  For FILESYSTEM users not on Windows or POSIX this will cause problems.

Should there be an XCHDIR or something like that?